### DISCUSSION TITLE
Any working status bar?

### DISCUSSION THREAD

**Discussion by wesbluemarine** [YaLTeR/niri#61] [Q&A] [answered]
I've tried waybar, hybrid-bar without success.

**Comment by YaLTeR** [comment, marked as answer]
Waybar works fine for me. Any layer-shell bar should work.

**Comment by wesbluemarine** [comment]
I had problems with xdg-desktop-portal; every gtk apps took a lot on startup.
This is my idea to have a clearly visible date without losing any space; waybar allows it to be used as an overlay over windows.
![image](https://github.com/YaLTeR/niri/assets/14043560/d6fb861e-2ab8-4d78-b1be-b1697b33f0c8)

**Reply by YaLTeR** [reply]
Ah, that's interesting.

**Reply by wesbluemarine** [reply]
I've noticed that with this waybar configuration the overlay remains also on top of full screen windows... Is there a solution to solve this?

**Reply by YaLTeR** [reply]
One solution is to put the bar on the bottom layer (rather than the default top).

I've been meaning to make it so fullscreen windows draw over the top layer-shell layer when they occupy the entire monitor, but haven't got to it yet.

**Reply by wesbluemarine** [reply]
With the bar on the bottom level the windows cover it while scrolling. I opted for a keyboard shortcut to toggle the bar.

`Mod+B { spawn "killall" "-SIGUSR1" "waybar"; }`

**Reply by YaLTeR** [reply]
In the commit I just pushed I made fullscreen windows render on top of the top layer when stationary (i.e. no animations ongoing).

**Reply by wesbluemarine** [reply]
It works! Thanks!
